### ISSUE TITLE
[Feature Request] Alliance Maps Delete Permissions

### ISSUE THREAD

**Issue by vlanf** [goryn-clade/pathfinder#187]
Follows https://github.com/exodus4d/pathfinder/issues/906

Any chance for implementing this feature?

**Comment by TyrHeimdalEVE** [comment]
It shouldn't be possible to delete the map unless you are a manager (requiring certain roles) or an admin (of the Pathfinder instance), if explicitly set in:
https://my.pathfinder.com/admin/settings

... or is my recollection of these a bit fuzzy?

**Comment by vlanf** [comment]
@TyrHeimdalEVE 

There is no settings for alliance maps, only corps. Anyone can create alliance map, and anyone can delete it.

**Comment by ElegosAkla** [comment]
I think this is because user / manager / admin role come from eve online character's corporation role.
and there is no character's alliance role.
for my instance if (I remember well) I fix this issues by disabling alliance creation map in page code.
(And I only made corporation map share with all my alliance corp.)

**Comment by TyrHeimdalEVE** [comment]
> There is no settings for alliance maps, only corps. Anyone can create alliance map, and anyone can delete it.

Like I said, recollection may be a bit fuzzy on this subject. ;-)

> I think this is because user / manager / admin role come from eve online character's corporation role.
and there is no character's alliance role.

Yeah the only way to solve it would be to assign the alliance executor corp the full ownership of the alliance, and take role permissions from that to evaluate if creating/deleting alliance maps should be possible.

And I'm not so certain it is worth the effort. Mostly everyone I know who uses Pathfinder, uses a corporation map and just adds alt corps if even needed.

With the addition of the `SESSION_SHARING` functionality, this becomes even less relevant for many as scanning toons can be in NPC corps and still get access by inheriting it from the main character (in corp).

And most wormhole groups rarely really need an alliance map. But there are those edge cases where bigger nullsec entities uses it.

> for my instance if (I remember well) I fix this issues by disabling alliance creation map in page code.

Shouldn't have to disable it in the code itself, just set MAX alliance maps to zero, and you don't have to fiddle with the codebase on updates to disable it again.

I will look into this however. In the original issue it says only managers can create alliance maps, which I can't recall being the case, but

**Comment by vlanf** [comment]
I agree, one can use corp map sharing feature instead, but it would be more convenient in our group to use alliance map.

>it says only managers can create alliance maps

Screenshot from regular user without permissions:

![image](https://github.com/goryn-clade/pathfinder/assets/25212107/5476c5cd-731a-46c6-baad-efc2055c4b2e)

>And I'm not so certain it is worth the effort. 

I do not insist to bring this feature to main branch, workaround with tweaking some code or db entry is ok for me.

P.S.

Also I want to notice that for some reason alliance map is missing in /admin/maps
